cicd/gitlab/parts/build-datasets-dual-deploy.gitlab-ci.yml

```diff
@@ -0,0 +1,90 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# License-related part of the Ensembl/ensembl-genomio CI/CD pipeline
+
+# A generic job to be used by all License jobs with the "extends" mechanism
+# (https://docs.gitlab.com/ee/ci/yaml/#extends)
+
+
+variables:
+  # https://docs.gitlab.com/ee/user/namespace/  
+  CONTAINER_IMAGE_NAME: datasets-cli
+  DATASETS_VERSION: "16.20.0"
+  DATASETS_DIR: containers/docker/datasets-cli
+  
+  # GITLAB:
+  GITLAB_PROJECT: vectorbase
+  GITLAB_REPO: ensembl-genomio
+  GITLAB_DOCKER_IMAGE: $GITLAB_PROJECT/$GITLAB_REPO/$CONTAINER_IMAGE_NAME
+  
+  # DOCKERHUB:
+  DOCKERHUB_PROJECT: ensemblorg
+  DHUB_DOCKER_IMAGE: $DOCKERHUB_PROJECT/$CONTAINER_IMAGE_NAME
+  DHUB_REG_URL: registry.hub.docker.com
+
+datasets_docker_build:
+  only:
+    variables:
+      - $DATASETS_CLI_SCHEDULER
+  stage: build
+  image: docker:latest
+  services:
+    - docker:dind
+
+  tags:
+    - dind
+  
+  script:
+    # Get the latest version of NCBI's datasets-cli software
+    # https://docs.gitlab.com/ee/ci/variables/predefined_variables.html    
+    - echo "Building $CONTAINER_IMAGE_NAME"
+    - echo "docker build -f ${DATASETS_DIR}/Dockerfile --build-arg datasets_version=$DATASETS_VERSION -t $CONTAINER_IMAGE_NAME $DATASETS_DIR"
```

Comment:
I wonder if it's possible to get url and version with, store them in the files
with something like
```
wget https://api.github.com/repos/ncbi/datasets/releases/latest -O - | grep browser_download_url | grep linux-amd64.cli.package.zip |  cut -f 4 -d '"' | tee container_url.txt | cut -f 8 -d / > container_ver.txt
```
and  use them to build the image or publish it